scripts/scmigrate/migrate.go

```diff
@@ -0,0 +1,151 @@
+package scmigrate
```

Comment:
May we have a [package comment](https://github.com/golang/go/wiki/CodeReviewComments#package-comments) please?